_posts/2021-03-20-discord.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: MDAnalysis Discord Server
+---
+
+
+We created the [{{ site.discord.name }} Discord]({{ site.discord.url
+}}) server for users and developers of MDAnalysis to talk and hang
+out, as an additional avenue to [participate]({{ site.baseurl
+}}/#participating) in our community.
+
+## Join us!
+
+In order to join, use the sign-up link [{{ site.discord.invite }}]({{
+site.discord.invite }}).
+
+By joining the MDAnalysis Discord Server you agree to our [Code of
+Conduct]({{ site.baseurl }}/pages/conduct/).
+
+
+<img src="{{ site.baseurl }}{{ site.images }}/discord-mdanalysis.svg"
+title="{{ site.discord.name }} Discord server (image used Discord icon
+by https://www.flaticon.com/authors/pixel-perfect)" alt="{{
+site.discord.name }} Discord" style="float: right; height: 12em; "/>
+
+## First steps
+
+We encourage you to introduce yourself in the *[#introductions]({{
+site.discord.url }}/807679126509060116)* channel.
```

Comment:
Do we want to ask folks to include they github handles and/or preferred pronouns?

Reply:
```suggestion
site.discord.url }}/807679126509060116)* channel. Please include anything you'd like us to know -- for example, if you have a preferred pronoun and or a GitHub account!
```

Some folks may not be comfortable pinning down their defined pronouns in a certain forum, so I usually default to "they".

Reply:
I agree with something like this - indicate we'll respect preferred pronouns but make it clear it's not expected for everyone to state theirs.

Reply:
It is very unlikely we will be able to keep track of people introducing themselves. If somebody writes about their preferred pronouns in their intro, how likely is it that we and the other members of the discord will (I) have seen it, (ii) remember about it? Keep in mind that there are very few members now but it that will change with time and as we promote the server. Also, you are paying specific attention to new members because of GSoC, now; how long will it last?

I am all about being welcoming, but I am a bit worried about promising anything like that on the behalf of everybody in the discord and getting people disappointed.

Reply:
> I am all about being welcoming, but I am a bit worried about promising anything like that on the behalf of everybody in the discord and getting people disappointed.

That's fair, since as you point out we can't really guarantee how everyone behaves. For me personally, it helps that Discord search could inform later interactions with them -- if you want to refer to them later or look up their account, the `from: username in: introductions` brings up their post in the introductions channel. We could add that to the blog post, like "That will help with chatting with other users, as you can look up their introductions by searching `from: username in: introductions`."

Reply:
Thanks for the search trick.